openassessment/templates/openassessmentblock/staff_area/oa_student_info_assessment_detail.html

```diff
@@ -0,0 +1,57 @@
+{% load i18n %}
+{% spaceless %}
+<div class="staff-info__status ui-staff__content__section wrapper--ui--collapse staff-info__{{ class_type }}__assessments">
+    <div class="ui-staff ui-toggle-visibility is--collapsed">
+        <h2 class="staff-info__title ui-staff__subtitle ui-toggle-visibility__control">
+            <i class="icon fa fa-caret-right" aria-hidden="true"></i>
+            <span>{% trans translated_title %}</span>
+        </h2>
+        <div class="ui-toggle-visibility__content">
+            {% for assessment in assessments %}
+                {% with assessment_num=forloop.counter %}
+                    {% if assessments|length > 1 %}
+                        <h4 class="title--sub">
+                            {% blocktrans %}
+                                Assessment {{ assessment_num }}:
+                            {% endblocktrans %}
+                        </h4>
+                    {% endif %}
+                    <table class="staff-info__status__table" summary="{% trans "Assessment" %}">
+                        <thead>
+                        <tr>
+                            <th abbr="Criterion" scope="col">{% trans "Criterion" %}</th>
```

Comment:
I know it was already this way, but I wonder if the abbr values should be translated.